app/managers.py

```diff
@@ -0,0 +1,56 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(
+            self
+    ) -> None:
+        self._sqlconnection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(
+            self,
+            first_name: str,
+            last_name: str
+    ) -> None:
+        self._sqlconnection.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._sqlconnection.commit()
+
+    def all(
+            self
+    ) -> list[Actor]:
+        actors_cursor = self._sqlconnection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
```

Comment:
use * to retrieve all columns from a table

Reply:
I don't think that's right.

https://mate.academy/en/learn/django-orm/django-orm-database-intro?section=video&testTaskSlug=py_actor_manager&videoId=2714
Time 5:13 

Danilo does the opposite to fix the bug of non-coretly creating class instances.:
From:
SELECT * FROM ...
To:
SELECT [colums_id, ....] FROM ...

![image](https://github.com/mate-academy/py-actor-manager/assets/142671/ae701f2b-5c09-4b57-af71-658d636a2d7c)